sherpa/utils/tests/test_utils.py

```diff
@@ -263,16 +264,16 @@ def test_neville2d(self):
                 val = utils.neville2d(xx, yy, x, y, fval)
                 self.assertTrue(utils.Knuth_close(answer, val, tol))
 
-    def test_parallel_map(self):
-        ncpus = multiprocessing.cpu_count()
 
+@pytest.mark.parametrize("num_tasks", [1, 3, 100])
+def test_parallel_map(num_tasks):
```

Comment:
I'm not happy with the function argument being called `num_tasks` as it is too similar to `numtasks` and makes it harder to see what is going on with the tests.